.typos.toml

```diff
@@ -0,0 +1,6 @@
+[default]
+extend-ignore-identifiers-re = [
+    "alle",
+    "groupe",
```

Comment:
These are `reserved` constants not found in the included wordbanks:

https://github.com/slackapi/slack-cli/blob/f3ba82430864d4434620c973bd9d089c19b1e979/internal/pkg/create/constants.go#L210-L225